src/error.rs

```diff
@@ -167,12 +181,24 @@ impl From<globset::Error> for Error {
   }
 }
 
+impl From<host_port_parse_error::HostPortParseError> for Error {
```

Comment:
Is there a call site which is invoking this conversion?

Reply:
Hmm---nope. It must've slipped in when I linearized the git history.